CHANGELOG.md

```diff
@@ -76,6 +76,10 @@ at velocity 0 it would look the same as its tail (but you can't have 0 velocity)
 
 ### MIDI
 
+#### <ins>MIDI_DEVICES SD card folder</ins>
+- A new folder has been created in the root of the SD card titled `MIDI_DEVICES`
+- The files `MIDIDevices.XML` and `MIDIFollow.XML` which were previously saved in the root of the SD card have been moved to the `MIDI_DEVICES` folder
```

Comment:
Do we want these in the same folder as midi device CC names? It seems like they'd be better off in a settings folder

Especially MIDIDevices.xml since it's not going to be obvious to users what it's for if they see it

Reply:
Good call, I'll move these + community features to a new DEFAULT_SETTINGS folder